### ISSUE TITLE
Showing progress during flux variability analysis.

### ISSUE THREAD

**Issue by achillesrasquinha** [opencobra/cobrapy#1279]
### Checklist

- [ ] There are [no similar issues or pull requests](https://github.com/opencobra/cobrapy/issues) for this yet.

## Describe the solution you would like.

The currently implementation of FVA within cobrapy is generally slow (definitely improved using the `processes` flag. Would be nice to see the current progress since the program sometimes seems stalled for large networks. What say? I was planning on using `tqdm` within `variability.py` at line 247.

e.g.

<img width="972" alt="Screen Shot 2022-09-19 at 2 57 36 PM" src="https://user-images.githubusercontent.com/9742568/191105608-3884ca5b-d34a-42c7-814c-2580e0d667b0.png">

**Comment by cdiener** [comment]
Good idea. We already have rich as a dependency, so would be better to use rich.progress.track for that. There should also be a toggle to turn that on and off, since some workloads may run an FVA for thousands of models and may not want a progress bar for each iteration. Or one could couple it to the logging level and only show it on INFO...

**Comment by achillesrasquinha** [comment]
I'll do that accordingly.

**Comment by achillesrasquinha** [comment]
@cdiener Done. PF Pull Request.
